src/RadixDlt.NetworkGateway.PostgresIntegration/DbQueryExtensions.cs

```diff
@@ -131,7 +161,12 @@ public static IQueryable<LedgerTransaction> GetLatestLedgerTransactionAtEpochRou
             .Take(1);
     }
 
-    public static IQueryable<LedgerTransaction> GetFirstLedgerTransactionAtEpochRound<TDbContext>(this TDbContext dbContext,  long epoch, long round)
+    /// <summary>
+    /// The long/horrible/verbose name is on purpose, to draw your attention and prevent people foot-gunning themselves.
+    /// A LedgerTransaction row contains large blobs, so you must SELECT the fields you need after using this, and not pull down the whole
+    /// ledger transaction row, to avoid possible performance issues.
+    /// </summary>
+    public static IQueryable<LedgerTransaction> GetFirstLedgerTransactionAtEpochRound__EnsureYouSelectRequiredFieldsAsRowsCanBeVeryBig<TDbContext>(this TDbContext dbContext,  long epoch, long round)
```

Comment:
this extension is used only in one place, maybe better to just inline it in plece where it's used?